src/current/v23.2/show-virtual-cluster.md

```diff
@@ -0,0 +1,127 @@
+
+## Required privileges
+
+`SHOW VIRTUAL CLUSTER` requires the `admin` role or the `MANAGEVIRTUALCLUSTER` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges).
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM MANAGEVIRTUALCLUSTER TO user;
+~~~
+
+{% include enterprise-feature.md %}
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/physical-replication/show-virtual-cluster-diagram.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+------------
+`virtual_cluster_spec` | The name of the virtual cluster.
+`REPLICATION STATUS` | Display the details of a replication stream.
+`CAPABILITIES` | Display the [capabilities]({% link {{ page.version.version }}/create-virtual-cluster.md %}#capabilities) of a virtual cluster.
+
+
+{% comment %}Responses and Data state are a copy from the monitoring page, these should be in an include once the first PR has merged.{% endcomment %}
+## Responses
+
+The following table is a list of all the responses returned with the different `SHOW VIRTUAL CLUSTER` statements:
+
+Field    | Response
+---------+----------
+`id` | The ID of the virtual cluster.
+`name` | The name of the standby (destination) virtual cluster.
+`data_state` | The state of the data on the virtual cluster. This can show one of the following: `initializing replication`, `ready`, `replicating`, `replication paused`, `replication pending cutover`, `replication cutting over`, `replication error`. Refer to [Data state](#data-state) for more detail on each response.
+`service_mode` | The service mode shows whether the virtual cluster is ready to accept SQL requests. This can show one of the following: <br>`none`<br>`shared`: The virtual cluster's SQL connections will be served by the same nodes that are serving the system interface.
+`source_tenant_name` | The name of the primary (source) virtual cluster.
+`source_cluster_uri` | The URI of the primary (source) cluster. This is the URI that connects to the primary cluster to [start a replication stream]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#step-4-start-replication).
+`replication_job_id` | The ID of the replication job.
+`replicated_time` | The latest timestamp at which the standby cluser has consistent data — that is, the latest time you can cut over to. This time advances while the replication proceeds without error. `replicated_time` is updated periodically (every `30s`).
+`retained_time` | The earliest timestamp at which the standby cluster has consistent data — that is, the earliest time you can cut over to.
+`cutover_time` | The time at which the cutover will begin.
```

Comment:
Is it worth mentioning that it. can be in the past or in the future?

Reply:
Done